nodes/parachain/BUILDING-DEB.md

```diff
@@ -0,0 +1,31 @@
+# Building .deb
```

Comment:
> I really like that guide! Since we have a `/docs` directory I would rather put the file there?

I could update the "docs" directory with the needed information about building the debs if that's OK? I'm still investigating if I could create a source deb somehow aswell since that would be even better. But as Rust packages with create.io is a major challange it might not be possible near term. I need to investigate more with some deb-pros.

Reply:
> It seems that deb-cargo needs to work from within a crate, yeah. It seems reasonable. I'm not sure if the current placement in the root is correct either. Makes more sense to have them with the crates for mashnet-node and kilt-parachain?

Yes you are right. It makes more sense to have them there. But AFAIK we decided against putting them there so that the docker build is faster. If only the dev-spec folder changed, we don't rebuild the project but only copy the json files over.

But the peregrine spec files are public now and also quite stable. So the best location would be to [bake it into the binary](https://github.com/KILTprotocol/mashnet-node/blob/b1431f464f3cdfdd8883cdfeaa50aa94b4755729/nodes/parachain/src/chain_spec/spiritnet.rs#L238). Also not 100% optimal but the best option I know IMO.